packages/core/src/components/split-button/split-button.scss

```diff
@@ -9,4 +9,15 @@
 
 :host {
   display: block;
```

Comment:
If you remove `btn-group` (inline-flex) from host element you must apply `inline-block` to :host selector if the component otherwise layout will take the complete line space